pom.xml

```diff
@@ -364,6 +350,29 @@
         <auto-value-annotation.version>1.6.6</auto-value-annotation.version>
```

Comment:
Isn't this up to 1.7 now?

Reply:
Good catch, I kept bouncing back and forth I've changed to 1.7.